DigitalAIOfficial/Workflows/AWS/Workflow_Create an AWS Lambda Function using S3 zip file.yaml

```diff
@@ -5,7 +5,7 @@ metadata:
   path: Workflows/AWS
   home: Digital.ai - Official/Workflows/AWS
 spec:
-- template: Create an AWS Lambda Function using S3 zip file
+- template: AWS Lambda create function using S3 zip file
```

Comment:
With change of template titles, it would be nice to also update filenames and its references.

Reply:
We have a ticket for next sprint, we will be updating the Template Titles, filenames and its references. will continue in this same pull request.

Reply:
We have a request for changing the plugin svg to have a unified look and feel, we have new logos from Bojan, 

Can we directly add it to the folder and update the yaml?

For example:
 file: !file "template-logo/451b3d960f1d50cbf645a535c781edf32783b4e1/**xlr-aws-plugin.svg**"